Gemfile

```diff
@@ -76,6 +76,7 @@ group :test do
 end
 
 group :development, :test do
+  gem 'database_cleaner'
   gem 'rspec-rails', '>= 3.9.0'
   gem 'shoulda-matchers', '~> 3.1', '>= 3.1.1'
 end
```

Comment:
- [ ] please make sure to add Capybara in your gemfile.

Reply:
@mikethreels  Capybara is present on line 73